statscache/app.py

```diff
@@ -36,24 +28,36 @@ def main(name):
     )
 
 
+@app.route('/')
+def index():
+    """ Generate a JSON-P response with an index of plugins (as an array) """
+    return jsonp(json.dumps(plugins.keys()), 200)
+
+
+@app.route('/<name>')
+def main(name):
+    """ Generate a JSON-P response with the content of the plugin's model """
+    callback = flask.request.args.get('callback')
+    status = 404
+    body = '"No such model for \'{}\'"'.format(name)
+    plugin = plugins.get(name)
+    if hasattr(plugin, 'model'):
+        model = plugin.model
+        status = 200
+        body = model.to_json(session.query(model).all())
```

Comment:
:+1: to that. *Definitely* noticed some slowness dumping everything in one response. Assuming neither of you already started coding, I'll get to work on integrating that code (thanks @rtnpro !).